heudiconv/utils.py

```diff
@@ -265,6 +265,32 @@ def json_dumps_pretty(j, indent=2, sort_keys=True):
     return js_
 
 
+def add_field_to_json(json_file, field_and_value):
+    """
+    Adds a given field (and its value) to a json file
+
+    Parameters:
+    -----------
+    json_file : str or Path
+        path for the corresponding json file
+    field_and_value : dict
+        pair of "key": "value" to add to the json file
+    """
+    for key, value in field_and_value.items():
+        lgr.debug(
+            'File: "{f}": Adding {k}: {v}'.format(
```

Comment:
```suggestion
            'File "{f}": Setting {k} to {v}'.format(
```
- since not necessarily adding, unless we do add some logic to at least warn or skip if record already exist (I have not  yet analyzed on where it is used)
- too many `:`s - would make it hard(er) to parse that log message